### PR TITLE
[issue-14] Allow to specify schema init script and schema name in housekeeping config

### LINKED ISSUE #14: Parametrize schema init script and schema name via housekeeping properties
Sometimes is useful to use the same RDMS to keep a set of HK databases for different processes. We'd like to be able to specify the schema name and the DB init script for each of these processes and potentially reuse the same server since these DB's tend to be small.

### PR DESCRIPTION
**Pull request by ddcprg**
Fixes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,14 +21,19 @@ In order to connect to your SQL database, you must place a database connector ja
 If your project utilises Spring YAML you can define your Housekeeping within the YAML. For example:
 
     housekeeping:
+      # Name of the schema/database to use -defaults to houseleeping
```
Comment (massdosage) [inline README.md]: -defaults to houseleeping 
->
- defaults to housekeeping

File: README.md [hunk 2]
```diff
@@ -21,14 +21,19 @@ In order to connect to your SQL database, you must place a database connector ja
 If your project utilises Spring YAML you can define your Housekeeping within the YAML. For example:
 
     housekeeping:
+      # Name of the schema/database to use -defaults to housekeeping 
+      schema-name: housekeeping
+      # Location of the script file to initialize the schema - defaults to classpath:/schema.sql 
+      db-init-script: housekeeping
```
Comment (patduin) [inline README.md]: did you mean to write 'housekeeping' here, or do you want to set the default value 'classpath:/schema.sql'?

Comment (ddcprg) [reply README.md]: no, good spot

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## [1.0.2] 2018-02-13
+### Changed
```
Comment (massdosage) [inline CHANGELOG.md]: NABD but this should be ### Added

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+* Add new housekeeping configuration parameters to set the schema name and the location of the DB initialization script.
```
Comment (massdosage) [inline CHANGELOG.md]: Add -> Added

File: README.md [hunk 5]
```diff
@@ -21,14 +21,19 @@ In order to connect to your SQL database, you must place a database connector ja
 If your project utilises Spring YAML you can define your Housekeeping within the YAML. For example:
 
     housekeeping:
+      # Name of the schema/database to use -defaults to housekeeping
```
Comment (massdosage) [inline README.md]: use -defaults -> use - defaults

Comment (ddcprg) [reply README.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (ddcprg) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (ddcprg) [reply CHANGELOG.md]: done

**Pull request merged**